test/common_utils.py

```diff
@@ -150,12 +150,9 @@ def accept_output(update_type):
                      "python {} {} --accept").format(munged_id, subname_output, output, __main__.__file__, munged_id))
 
         if ACCEPT:
-            equal = False
             try:
-                equal = self.assertEqual(output, expected, prec=prec)
+                self.assertEqual(output, expected, prec=prec)
             except Exception:
-                equal = False
-            if not equal:
                 return accept_output("updated output")
```

Comment:
I have an impression that `return accept_output("updated output")` is always returned ?

Reply:
@vfdev-5 Absolutely, the return can go away. Also the `accept_output()` does not return either. Let me make a correction.

Reply:
@datumbox thinking more about this test, I think your code is correct with return in `except` branch. I think the intention is to update the output if `output != expected`.

Reply:
@vfdev-5 I saw you approved but could you please have another look? I made some more changes.